src/poetry/core/masonry/builders/builder.py

```diff
@@ -48,21 +48,15 @@ def _module(self) -> Module:
 
         packages: list[dict[str, str | dict[str, str]]] = []
         includes: list[dict[str, str | dict[str, str]]] = []
-        for source_list, target_list, default in [
-            (self._package.packages, packages, ["sdist", "wheel"]),
-            (self._package.include, includes, ["sdist"]),
+        for source_list, target_list in [
+            (self._package.packages, packages),
+            (self._package.include, includes),
         ]:
             for item in source_list:
-                # Default to including in both sdist & wheel
```

Comment:
We should drop this as well. Seems to be a left over from [1bdbd900c4fc1e2d4ed1ef15dfbcafdcb66dd892](https://github.com/python-poetry/poetry-core/commit/1bdbd900c4fc1e2d4ed1ef15dfbcafdcb66dd892). 

This is no longer accurate. We are defaulting to `sdist` for `includes` and both for `packages`.